README.md

```diff
@@ -42,44 +42,44 @@ Provides a higher level API for command-line argument parsing than interacting
 with `process.argv` directly. Takes a specification for the expected arguments
 and returns a structured object with the parsed options and positionals.
 
+For example, assuming the following script for `simple.js`:
+
 ```mjs
 import { parseArgs } from 'node:util';
-const args = ['-f', '--bar', 'b'];
 const options = {
-  foo: {
-    type: 'boolean',
-    short: 'f'
-  },
-  bar: {
-    type: 'string'
-  }
+  foo: { type: 'boolean', short: 'f' },
+  bar: { type: 'string' }
 };
-const {
-  values,
-  positionals
-} = parseArgs({ args, options });
-console.log(values, positionals);
-// Prints: [Object: null prototype] { foo: true, bar: 'b' } []
+try {
+  const { values } = parseArgs({ options });
+  console.log(values);
+} catch (err) {
+  console.log(`${err.code}: ${err.message}`);
+}
 ```
 
 ```cjs
 const { parseArgs } = require('node:util');
-const args = ['-f', '--bar', 'b'];
 const options = {
-  foo: {
-    type: 'boolean',
-    short: 'f'
-  },
-  bar: {
-    type: 'string'
-  }
+  foo: { type: 'boolean', short: 'f' },
+  bar: { type: 'string' }
 };
-const {
-  values,
-  positionals
-} = parseArgs({ args, options });
-console.log(values, positionals);
-// Prints: [Object: null prototype] { foo: true, bar: 'b' } []ss
+try {
+  const { values } = parseArgs({ options });
+  console.log(values);
+} catch (err) {
+  console.log(`${err.code}: ${err.message}`);
```

Comment:
I think typical usage in simple scripts won't bother with the try-catch.

If you want to keep the try-catch, this should be `console.error`, and probably should be followed by a `process.exit(1)`.

Reply:
Removed error handling and example.